code_examples/sdk_examples/src/core_features/web3names/04_release.ts

```diff
@@ -2,7 +2,7 @@ import * as Kilt from '@kiltprotocol/sdk-js'
 
 export async function releaseWeb3Name(
   did: Kilt.DidUri,
-  submitterAccount: Kilt.KiltKeyringPair,
+  submitterKeyPair: Kilt.KiltKeyringPair,
```

Comment:
What's the reason for this change? It's called `submitterAccount` to distinguish it from the DID and emphasize that it's the keypair of an account.

Reply:
I can revert this, but I felt it that name would be clearer, since this is not only account info, but also its private key. But I see that we should be consistent with this, since we also use "submitterAccount" often elsewhere.

Reply:
Yeah either rename it everywhere or don't. I'm still in favour of not renaming it. But I also see the argument for calling it `submitterKeyPair`. So it's up to you...

Reply:
reverted.